packages/react-components/react-divider/src/components/Divider/Divider.types.ts

```diff
@@ -7,7 +7,7 @@ export type DividerSlots = {
   root: NonNullable<Slot<'div'>>;
 
   /**
-   * Accessibility wrapper for content when presented.
+   * Wrapper for content when presented.
```

Comment:
Chatted w/ Bernardo about this, and confirmed wrapper isn't used for a11y. While it does function as the label for the divider, it'd be equally effective to use a self-referencing `aria-labelledby`, so `wrapper` is kinda incidental from an a11y perspective.